lib/tailwind.ex

```diff
@@ -170,6 +170,7 @@ defmodule Tailwind do
 
     with true <- File.exists?(path),
          {out, 0} <- System.cmd(path, ["--help"]),
+         out <- Regex.replace(~r/(\x9B|\x1B\[)[0-?]*[ -\/]*[@-~]/, out, ""),
```

Comment:
Can you please try this instead?

```suggestion
         {out, 0} <- System.cmd(path, ["--help"], env: %{"NO_COLOR" => "1"}),
```

And let me know if it works!

Reply:
that code works in my environment very well. so, I amend commit and re-push.